readme/LinuxBuild.md

```diff
@@ -105,7 +106,18 @@ TODO
 
 #### AppImage
 
-TODO
+One way to generate an AppImage is to first generate the `.tar.gz` package and
+then use that as the `AppDir` for `linuxdeploy`. For an example, see
+[`azure-pipelines/release.yml`](../azure-pipelines/release.yml)
+
+```bash
+cmake .. -DPACK_GENERATOR="TGZ"
```

Comment:
Why not just use `make install DESTDIR=my.AppDir`? This seems a little too complex. Perhaps I'm missing something.

Reply:
The install target will actually install to `$DESTDIR/${CMAKE_INSTALL_PREFIX}`, which can cause problems if the user has configured cmake with `-DCMAKE_INSTALL_PREFIX=...`. The tarball seems to "just work" and I haven't found a better solution yet.

I also need to update the instructions here to use the `build_appimage.sh` script...